devel/ldid-procursus/Portfile

```diff
@@ -0,0 +1,34 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                ldid-procursus
+version             2.1.5-procursus7
+categories          devel
+license             AGPL-3
+platforms           darwin
+conflicts           ldid
+maintainers         @therealketo procurs.us:team
+description         Put real or fake signatures in a Mach-O.
+long_description    ${description}
+homepage            https://github.com/ProcursusTeam/ldid
+
+github.setup        ProcursusTeam ldid ${version} v
+github.tarball_from archive
+checksums           rmd160 648697e84ac2b71fff24fdc1241f90d7390124d5 \
+                    sha256 04e461c6f02765e48fc9cc0b68d4dc353a9c46bc1c4d8bac0695509d1af1ff5e \
+                    size 53532
+
+depends_build       port:pkgconfig
+depends_lib         path:lib/libplist-2.0.dylib:libplist \
+                    path:lib/libcrypto.3.dylib:openssl
+
+use_configure       no
+universal_variant   no
+use_xcode           no
+destroot.args       PREFIX=${prefix}
```

Comment:
I feel that you're looking for PortGroup `makefile`, am I wrong?

Reply:
The project only uses a Makefile. How would I go about configuring the Portfile so that it doesn't assume an `autotools` setup?

Reply:
I mean something like:
```
diff --git a/devel/ldid-procursus/Portfile b/devel/ldid-procursus/Portfile
index 0089994cb39..6fba7a4ba40 100644
--- a/devel/ldid-procursus/Portfile
+++ b/devel/ldid-procursus/Portfile
@@ -1,5 +1,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           makefile 1.0
+PortGroup           openssl 1.0
 
 name                ldid-procursus
 version             2.1.5-procursus7
@@ -18,14 +20,9 @@ checksums           rmd160 648697e84ac2b71fff24fdc1241f90d7390124d5 \
                     sha256 04e461c6f02765e48fc9cc0b68d4dc353a9c46bc1c4d8bac0695509d1af1ff5e \
                     size 53532
 
-depends_build       port:pkgconfig
-depends_lib         path:lib/libplist-2.0.dylib:libplist \
-                    path:lib/libcrypto.3.dylib:openssl
+depends_lib-append  path:lib/libplist-2.0.dylib:libplist
 
-use_configure       no
 universal_variant   no
-use_xcode           no
-destroot.args       PREFIX=${prefix}
 
 post-destroot {
     set zsh_comp_path ${destroot}${prefix}/share/zsh/site-functions
```

What dramatically simplified portfile.

Reply:
Yea, the project doesn't have much of a build system, lol